src/test/java/ladder/domain/line/LineGeneratorTest.java

```diff
@@ -0,0 +1,171 @@
+        int width = 1;
+        int firstIndex = 0;
+        LineGenerator lineGenerator = new LineGenerator(new DirectionGeneratorImpl());
+        Line line = lineGenerator.generate(width);
+        List<Direction> directionsInfo = line.getDirectionsInfo();
+
+        //when
+        Direction direction = directionsInfo.get(firstIndex);
+
+        //when, then
+        assertThat(direction).isNotEqualTo(Direction.LEFT);
+    }
+
+    @DisplayName("라인의 두번째 값 부터는 이전의 값이 '오른쪽'이라면 다음 값은 무조건 '왼쪽'이 와야 한다.")
+    @Test
+    void lineGenerateTestByRightBeforeLeft() {
+        //given
+        int width = 2;
+        int firstIndex = 0;
+        int secondIndex = 1;
+        LineGenerator lineGenerator = new LineGenerator(new DirectionRightGeneratorImpl());
+        Line line = lineGenerator.generate(width);
+        List<Direction> directionsInfo = line.getDirectionsInfo();
+
+        //when
+        Direction firstLineDirection = directionsInfo.get(firstIndex);
+        Direction secondLineDirection = directionsInfo.get(secondIndex);
+
+        //when, then
+        assertThat(firstLineDirection).isEqualTo(Direction.RIGHT);
+        assertThat(secondLineDirection).isEqualTo(Direction.LEFT);
+    }
+
+    @DisplayName("라인에서 두번째 값 부터는 이전 값이 '왼쪽'이라면 다음 값은 '오른쪽' 또는 '중립'이 되어야 한다.")
+    @Test
+    void lineGenerateTestByRNBeforeLeft() {//수정 필요
+        //given
+        int width = 3;
+        int secondIndex = 1;
+        int thirdIndex = 2;
+        LineGenerator lineGenerator = new LineGenerator(new DirectionRightGeneratorImpl());
+        Line line = lineGenerator.generate(width);
+        List<Direction> directionsInfo = line.getDirectionsInfo();
+
+        //when
+        Direction secondLineDirection = directionsInfo.get(secondIndex);
+        Direction thirdLineDirection = directionsInfo.get(thirdIndex);
+        boolean isDirectionRN = (thirdLineDirection == Direction.RIGHT
+                || thirdLineDirection == Direction.NEUTRAL);
+
+        //when, then
+        assertThat(secondLineDirection).isEqualTo(Direction.LEFT);
+        assertThat(isDirectionRN).isTrue();
+    }
+
+    @DisplayName("라인에서 두번째 값 부터는 이전 값이 '중립'이라면 다음 값은 '오른쪽' 또는 '중립'이 되어야 한다.")
+    @Test
+    void lineGenerateTestByRNBeforeNeutral() {
+        //given
+        int width = 2;
+        int firstIndex = 0;
+        int secondIndex = 1;
+        LineGenerator lineGenerator = new LineGenerator(new DirectionNeutralGeneratorImpl());
+        Line line = lineGenerator.generate(width);
+        List<Direction> directionsInfo = line.getDirectionsInfo();
+
+        //when
+        Direction firstLineDirection = directionsInfo.get(firstIndex);
+        Direction secondLineDirection = directionsInfo.get(secondIndex);
+        boolean isDirectionRN = (secondLineDirection == Direction.RIGHT
+                || secondLineDirection == Direction.NEUTRAL);
+
+        //then
+        assertThat(firstLineDirection).isEqualTo(Direction.NEUTRAL);
+        assertThat(isDirectionRN).isTrue();
```

Comment:
여러 값으로 나타낼 수 있는 값에 조금이라도 로직을 추가하면, 설령 그게 == 밖에 없어도 읽을 때 어려워져요.
특히 본인이 작성한 로직이 아니라면 더 그렇죠!
assertJ 는 꽤나 잘 만든 라이브러리라서 이런 상황을 대비한 메서드가 항상 있는데요, 한 번 공식문서 읽어보며 학습한 뒤에 다양하게 활용해보는 것은 어떨까요? assertJ 공식문서는 쉽고 나름 재밌어서 공식문서 입문용으로도 아주 좋아요 😃 

```suggestion
        //when
        Direction secondLineDirection = directionsInfo.get(secondIndex);
        Direction thirdLineDirection = directionsInfo.get(thirdIndex);

        //when, then
        assertThat(secondLineDirection).isEqualTo(Direction.LEFT);
        assertThat(thirdLineDirection).isIn(Direction.RIGHT, Direction.NEUTRAL);
```

Reply:
해당 코멘트가 숨겨져 있어서 이제 봤네요😥
assertJ를 더 자연스럽게 사용하기 위해서 학습해봐야겠어요!